VTEX - Logistics API.json

```diff
@@ -3836,50 +3828,50 @@
                                                 "description": "Transfer."
```

Comment:
what does this transfer mean? not clear given the description and the example

Reply:
That description with the single word ("Transfer.") was already in the API, but the next endpoints to be reviewed are related to transfer supply lots! All the other mentions to the field "transfer" are currently just that one word, I will improve them all after doing some more research.